cfme/tests/containers/test_smart_management.py

```diff
@@ -42,30 +43,37 @@ def get_object_name(obj):
 
 
 def set_random_tag(instance):
+    logger.info("Setting random tag")
     navigate_to(instance, 'Details')
     toolbar.select('Policy', 'Edit Tags')
 
     # select random tag category
     cat_selector = AngularSelect("tag_cat")
     random_cat = random.choice(cat_selector.all_options)
+    logger.info("Selected category {cat}".format(cat=random_cat))
```

Comment:
Maybe ``logger.debug``?

Reply:
OPTIONAL: named fields for ``format`` are useful when there are several fields. When there's just one, they don't improve readability, IMHO. I would just use ``"Selected category {}".format(random_cat)``. Applies to other strings as well.

Reply:
The name in the curly brackets  make the code more readable in my opinion